src/Tests/Tests.Core/ManagedElasticsearch/NodeSeeders/DefaultSeeder.cs

```diff
@@ -36,22 +39,46 @@ public DefaultSeeder(IElasticClient client, IIndexSettings indexSettings)
 		public void SeedNode()
 		{
 			if (!TestClient.Configuration.ForceReseed && this.AlreadySeeded()) return;
-			// Ensure a clean slate by deleting everything regardless of whether they may already exist
-			this.ClusterSettings();
-			this.DeleteIndicesAndTemplates();
-			// and now recreate everything
-			this.CreateIndicesAndSeedIndexData();
+
+			var t = Task.Run(async () => await this.SeedNodeAsync());
+
+			t.Wait(TimeSpan.FromSeconds(40));
+		}
+		public void SeedNodeNoData()
+		{
+			if (!TestClient.Configuration.ForceReseed && this.AlreadySeeded()) return;
+
+			var t = Task.Run(async () => await this.SeedNodeNoDataAsync());
+
+			t.Wait(TimeSpan.FromSeconds(40));
 		}
```

Comment:
Should we consider removing both sync methods in favour of making `SeedCluster()` async on the cluster?

Reply:
We should! Wanted to keep this small before the 6.3.0 release